.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: python
+python:
+  - "2.7"
+  - "3.6"
```

Comment:
I think you want to remove one of these. Otherwise it will run the tests twice, but they'll both use the same python version: the one listed in the dockerfile.